src/accessory.ts

```diff
@@ -18,6 +18,7 @@ const displayName = (string: string): string => {
 };
 
 export interface UserConfig {
+  tvName?: string;
```

Comment:
given that we are overriding `friendlyName` what about `friendlyName` here too for consistency ?

Reply:
Changed tvName to friendlyName here and in schema, but left title attribute in schema as "TV Name"